sdk/include/psdk/winnls.h

```diff
@@ -180,6 +180,10 @@ extern "C" {
 #define NORM_IGNOREWIDTH	131072
 #define LINGUISTIC_IGNORECASE 0x00000010
 #define NORM_LINGUISTIC_CASING 0x08000000
+#define FIND_STARTSWITH 0x00100000
```

Comment:
```suggestion

#if (WINVER >= 0x0601)
#define FIND_STARTSWITH 0x00100000
```

Reply:
What is the point of restricting ourself like this? You can add a comment noting which Windows version it was added in but kernel32 should be allowed to use this flag inside itself in any target version IMHO.

Or something like `#if (WINVER >= 0x0601) || MODULE_KERNEL32` if we could get cmake to add a define like this for us.

Reply:
Note that these are public headers, so having ROS-specific ifdefs inside isn't really welcome.